design_practice/src/assets/stylesheets/shared/buttons.sass

```diff
@@ -0,0 +1,39 @@
+.button
+  background-color: $gray-98
+  border-radius: $border-radius
+  border: solid 1px $gray-75
+  color: $gray-30
+  font-size: $font-size-xxs
+  font-weight: 800
+  padding: $m-sm
+  height: 32px
+  text-align: center
+  min-width: 100px
+  text-decoration: none
+
+  &:focus
+    outline: none
+
+  &:hover
+    +media-touch-devices
+      background-color: inherit !important
+      color: inherit !important
+
+  &.-primary
+    animation: 1s infinite linear pulse
+    background-image: radial-gradient($primary-gradient-1, $primary-gradient-2)
+    border-radius: 3px
+    border: 0
+    color: white
+
+    &:hover
+      opacity: 0
+      cursor: none
+      +animate(opacity)
```

Comment:
Mixins should go first. Also you usually use them before the `hover` action, to make sure the animation also occurs when moving the mouse outside the button